rmake.py

```diff
@@ -268,13 +268,10 @@ def config_cmd():
     if args.compiler is not None:
         cmake_options.append(f"-DCMAKE_CXX_COMPILER={args.compiler}")
 
-    # windows defaults to clang++ in toolchain
+    # amdclang++ default in linux toolchain, clang++ in windows toolchain
     if os.name != "nt" and args.compiler is None:
         if args.use_hipcc_compiler:
             cmake_options.append(f"-DCMAKE_CXX_COMPILER={rocm_path}/bin/hipcc")
```

Comment:
This is the line you need to change.  Probably should change use_hipcc_compiler to use_amdclang_compiler?   If they want to actually use hipcc they can use --compiler flag